Orange/widgets/widget.py

```diff
@@ -205,6 +205,8 @@ def __new__(cls, *args, captionTitle=None, **kwargs):
 
         sc = QShortcut(QKeySequence.Copy, self)
         sc.activated.connect(self.copy_to_clipboard)
+        if self.controlArea is not None:
+            self.controlArea.setFocus()  # Otherwise, the first control has focus
```

Comment:
OK, I added `Qt.ActiveWindowFocusReason`. It probably doesn't change anything, though.